poc_mobile_verifier/src/speedtests.rs

```diff
@@ -403,22 +401,27 @@ impl SpeedtestTier {
 // we need is fetch from an actual database and mock fetch that returns
 // nothing.
 
+#[derive(thiserror::Error, Debug)]
+#[error(transparent)]
+pub struct FetchError(#[from] sqlx::Error);
+
 #[async_trait::async_trait]
 pub trait SpeedtestStore {
-    async fn fetch(self, id: &PublicKey) -> Result<Option<SpeedtestRollingAverage>>;
+    async fn fetch(self, id: &PublicKey) -> Result<Option<SpeedtestRollingAverage>, FetchError>;
```

Comment:
why the special error struct here but we just used a `sqlx::Error` in the `heartbeats.rs` module?

Reply:
mistake